pkg/telemetry/config.go

```diff
@@ -0,0 +1,122 @@
+package telemetry
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"text/template"
+	"time"
+
+	"github.com/warpbuilds/warpbuild-agent/pkg/log"
+)
+
+const (
+	configFilePath              = "/runner/warpbuild-agent/pkg/telemetry/otel-collector-config.yaml"
+	configTemplatePath          = "/runner/warpbuild-agent/pkg/telemetry/otel-collector-config.tmpl"
+	otelCollectorOutputFilePath = "/runner/warpbuild-agent/pkg/telemetry/otel-out.log"
+	binariesDir                 = "/runner/warpbuild-agent/pkg/telemetry/binaries"
```

Comment:
For macos we are keeping things inside `~/warpbuild/` I think, move if you can (not an issue), check that there are no permission issues btw.

Reply:
Yes. Changed this to take base path from the config itself.